app/dashboard/migrations/0051_auto_20180401_0947.py

```diff
@@ -0,0 +1,154 @@
+# Generated by Django 2.0.3 on 2018-04-01 16:47
+
+from django.db import migrations
+
+def insert_tools(apps, schema_editor):
+    Tool = apps.get_model('dashboard', 'Tool')
+
+    #basics
+    issue_explorer = Tool(name="Issue Explorer", category="BASIC", img="v2/images/why-different/code_great.png", description='''A searchable index of all of the funded work available in
```

Comment:
For example this is not translated anymore as before, I wonder how it should work. 
I can add to model method _name which will call gettext_lazy.

Reply:
maybe we can just run each user-facing attribute of the model through `gettext_lazy` before it hits the view...


edit: i think youre saying the same thing as me! carry on :)

Reply:
@kziemianek You should be able to use `trans` or `blocktrans` template tags in the `toolbox.html` template itself.

Reply:
I moved translation code to model, hope it's fine, but it really smells like view layer thing.

Reply:
🙌 That works!